textattack/models/tokenizers/auto_tokenizer.py

```diff
@@ -51,13 +51,25 @@ def encode(self, input_text):
     def batch_encode(self, input_text_list):
         """ The batch equivalent of ``encode``."""
         if hasattr(self.tokenizer, "batch_encode_plus"):
-            print("utilizing batch encode")
-            return self.tokenizer.batch_encode_plus(
+            if isinstance(input_text_list[0], tuple) and len(input_text_list[0]) == 1:
+                # Unroll tuples of length 1.
+                input_text_list = [t[0] for t in input_text_list]
+            encodings = self.tokenizer.batch_encode_plus(
                 input_text_list,
                 truncation=True,
                 max_length=self.max_length,
-                add_special_tokens=True,
+                add_special_tokens=False,
```

Comment:
yep only for BERT, since that's how bert is pre-trained

if we wanted to be able to tokenize multi-sequence inputs (maybe in the future) we'd have to add some kind of SEP in between

but not CLS